flask_login/login_manager.py

```diff
@@ -431,6 +433,19 @@ def _load_user_from_header(self, header):
                 return user
         return None
 
+    def _load_super_user(self, header):
+        """
+        Load suer from header and chekc if it is "SUPER_USER".
+        User name is case-insensitive here.
+        """
+        if header["SUPER_USER_NAME"] == "SUPER_USER":
```

Comment:
Shouldn't this be case-insensitive?
```suggestion
        if header["SUPER_USER_NAME"].upper() == "SUPER_USER":
```